web/src/components/pages/login.test.js

```diff
@@ -0,0 +1,23 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import ConnectedLogin from '../../containers/login';
+import { getStore } from '../../tests/testUtils';
+
+test('Login button is disabled without input values', () => {
+    const initialState = {
```

Comment:
minor: we try to use 2 space indent in the rest of the files, would you mind changing this here as well?

Additionally, to avoid that in the future, we could also add the `"indent": ["error", 2]` rule to `.eslintrc.json`. What do you think?

Reply:
My attempts to add an indent level broke and I couldn't figure out how to get around it. Build spinner-button continued to spin until manually terminated 
`HH@Jessies-MacBook-Pro$ (git::setup_jest_enzyme) npm start

> sentence-collector@1.12.0 start /Users/HH/personal/sentenceCollector/sentence-collector
> npm run clean && npm run env parcel web/index.pug


> sentence-collector@1.12.0 clean /Users/HH/personal/sentenceCollector/sentence-collector
> rm -rf dist .cache


> sentence-collector@1.12.0 env /Users/HH/personal/sentenceCollector/sentence-collector
> env $(grep -v '^#' .env | xargs) "parcel" "web/index.pug"

Server running at http://localhost:1234 
⠹ Building config.js...(node:73827) UnhandledPromiseRejectionWarning: TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Node'
    |     property 'local' -> object with constructor 'Node'
    --- property 'parent' closes the circle
    at JSON.stringify (<anonymous>)
    at process.target._send (internal/child_process.js:778:25)
    at process.target.send (internal/child_process.js:676:19)
    at Child.send (/Users/HH/personal/sentenceCollector/sentence-collector/node_modules/@parcel/workers/src/child.js:32:13)
    at Child.handleRequest (/Users/HH/personal/sentenceCollector/sentence-collector/node_modules/@parcel/workers/src/child.js:67:10)
(node:73827) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:73827) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
⠴ Building spinner-button.css...`

Reply:
This seems to have worked in the latest iteration, thanks! Could you also adjust testUtils.js?